src/Abc.Zebus/SubscriptionHandling/SubscriptionHandler.cs

```diff
@@ -0,0 +1,27 @@
+using Abc.Zebus.Directory;
+
+namespace Abc.Zebus.SubscriptionHandling
+{
+    /// <summary>
+    /// Extend this class with the message you wish to be notified of subscriptions for.
+    /// </summary>
+    /// <typeparam name="TMessage">The message that you wish to be notified for.</typeparam>
+    public abstract class SubscriptionHandler<TMessage> : IMessageHandler<SubscriptionUpdatedMessage>
```

Comment:
Why not an interface instead?

Reply:
Because we want to filter the messages that are handled